src/config.js

```diff
@@ -3,7 +3,9 @@ const config = {
     port: 3000,
   },
   grpc: {
-    nativePort: 3010,
```

Comment:
Why did we remove the native port? Did we decide to not use native connection anymore?